src/cpp/types/JSONTags.cpp

```diff
@@ -0,0 +1,86 @@
+// Copyright 2021 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <fastdds-statistics-backend/types/JSONTags.h>
+
+namespace eprosima {
+namespace statistics_backend {
+
```

Comment:
I think this should be independent from the database, as it will be needed for the user to read the json files, and it does not require to know that a database or database namespace exists.